src/components/controllers/repl-page.jsx

```diff
@@ -38,42 +38,31 @@ export default function ReplPage() {
  *
  * ?code -> ?example -> localStorage -> simple counter example
  */
-function initialCode(query) {
-	let code, slug;
```

Comment:
TODO: Split this out. Needs to land still, reduces the mess, but not necessarily needed here.